<?xml version="1.0"?>

<launch>

  <node pkg="traj_generator" type="circle_test" name="circle_test" output="screen">
    <param name="radius" type="double" value="1.0"/>
    <param name="period" type="double" value="6.0"/>
    <param name="rev" type="double" value="1.0"/>
    <param name="follow_yaw" type="bool" value="false"/>
    <param name="move_z" type="bool" value="false"/>
    <param name="z_diff" type="double" value="0.3"/>

    <!-- <remap from="~odom" to="/pos_vel_mocap/odom_TA" /> -->
    <!-- <remap from="~odom" to="/vins_multi_rgbd/imu_propagate" /> -->
    <remap from="~odom" to="/ekf/ekf_odom" />

  </node>

</launch>
